### PR TITLE
Server side

### PR DESCRIPTION
**Pull request by kpresun**
Please review my route put and delete on server side

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 const express = require('express');
 const app = express();
-const bodyParser = require('body-parser');
+
 const PORT = process.env.PORT || 5000;
-const koalaRouter = require('./routes/koala.router')
+const koalaRouter = require('./routes/koala.router');
+const { query } = require('express');
```
Comment (rubioz310) [inline server/server.js:6]: What is this for?

File: server/routes/koala.router.js [hunk 2]
```diff
@@ -1,18 +1,79 @@
...
+    console.log('request router parameters', req.params);
+    const koalaID= req.params.id;
+    
+    const queryText = `DELETE FROM koalas WHERE id = $1;`;
+    pool.query(queryText, [koalaID.id])
+
+    })
+    .then((response) => {
+        console.log('Deleted koala from dataset', ${response.rowCount ===1});
+        res.sendStatus(201);
```
Comment (rubioz310) [inline server/routes/koala.router.js:72]: Look for status when deleting

File: server/public/scripts/pool.js [hunk 3]
```diff
@@ -14,4 +25,5 @@ pool.on("connect", () => {
 pool.on("error", error => {
     console.log("Error with postgresql tool", error);
 });
-module.exports = pool;
\ No newline at end of file
+module.exports = pool;
+>>>>>>> 05bcc5c026a84e7cc704e609d4d7be808a9c3d4f
```
Comment (rubioz310) [inline server/public/scripts/pool.js:29]: Do you see this in your code?

File: server/routes/koala.router.js [hunk 4]
```diff
@@ -1,18 +1,79 @@
...
+      console.log(`error making query ${queryText}`, error);
+    });
+  })
+  
 // POST
 
+router.post('/', (req, res) => {
+    console.log('Inside router.post', req);
+    const koalaData = req.body;
+    const queryText = `INSERT INTO koalas ( name, gender, age, readyToTransfer, note)
```
Comment (rubioz310) [inline server/routes/koala.router.js:26]: Make sure is 'note' and not 'notes'

Comment (kpresun) [reply server/routes/koala.router.js:26]: changed to 'notes'

**Pull request merged**